cmd/thanos/receive.go

```diff
@@ -318,6 +321,7 @@ func runReceive(
 		if cw != nil {
 			ctx, cancel := context.WithCancel(context.Background())
 			g.Add(func() error {
+				// TODO: If config is empty we never start receiver. We might want to fail in this case.
```

Comment:
can we open an issue about this? :)

Reply:
sure (:

Reply:
Done: https://github.com/thanos-io/thanos/issues/2160